src/main/webapp/js/maps-page-loader.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * To style the map
+ */
+
+function createMap() {
+
+	// Create a new StyledMapType object, passing it an array of styles,
+	// and the name to be displayed on the map type control.
+	var styledMapType = new google.maps.StyledMapType(
+			[
+				{
```

Comment:
This whole array seems like it ideally would be defined in another static file (like a .json file) as it is basically a configuration. However, that's too much work for now and is not necessary. But just fyi, usually we try to separate dynamic logical code and static configurations, and they are actually considered treated very differently.